### PR TITLE
`Development`: Add screencast links to docs

### PR DESCRIPTION
**Pull request by pellumbbaboci**
#### General

- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/#naming-conventions-for-github-pull-requests).

### Description

Included screencast links to documentation.

#### Code Review
- [ ] Code Review 1
- [ ] Code Review 2
#### Manual Tests
- [ ] Test 1
- [ ] Test 2

### CODE REVIEW

File: docs/user/lectures.rst [hunk 1]
```diff
@@ -46,14 +46,22 @@ Instructors have three options for creating a lecture.
...
     Instructors can also customize the table entries by removing or adding new units. Additionally, they can enter keywords if they want particular slides
     to be removed. Slides containing one of the comma-separated keywords will be removed.
 
        |processed-lecture-units|
 
+    The following video shows how to split a lecture PDF into multiple units.
+
+    .. raw:: html
+
+        <iframe src="https://live.rbg.tum.de/w/artemisintro/39407?video_only=1&t=0" allowfullscreen="1" frameborder="0" width="600" height="350">
```
Comment (Hialus) [inline docs/user/lectures.rst:61]: At least for me this is just the same video as 39409 above and talks about referencing single slides. Could you check this please?

**Review by Hialus — CHANGES_REQUESTED** [review]

File: docs/user/lectures.rst (hunk 1 above)
Comment (pellumbbaboci) [reply docs/user/lectures.rst:61]: Users can reference single slides if instructors have uploaded lecture attachment units. So this is intentional, after the units are uploaded, this video is showing how to reference single slide.

Comment (pellumbbaboci) [reply docs/user/lectures.rst]: now it should be fixed 👍

**Review by Hialus — APPROVED** [review]
Video links are fixed now.
I looked through them and they look good! 🚀 
But in case you want/have to reshoot them I'd have some optional suggestions how you could improve them further:
- The keyboard clicking sounds are quite distracting imo
- There are quite a few "uhm"s. Its always great if these can be removed
- When you switch between contexts (student, instructor) the theme of Artemis changes. I'd try to keep the light mode on both and add a note in one of the corners, which context you are in currently

**Review by lennart-keller — APPROVED** [review]
Videos look good to me overall. I agree to Timors optional suggestions, but the videos are clear and understandable nonetheless :)

**Review by egekurt123 — APPROVED** [review]
Videos look generally good. On top of Timor´s suggestions, increasing the sound volume maybe would be a great idea too.

**Pull request merged**